pom.xml

```diff
@@ -14,7 +14,7 @@
     <name>online-book-store</name>
     <description>online-book-store</description>
     <properties>
-        <java.version>21</java.version>
+        <java.version>20</java.version>
```

Comment:
why java 20? it's not LTS (long time support) release, I'd suggest switching to either 17 or 21 (preferably 17, it's in task descirption)